tools/cli/commands.js

```diff
@@ -3107,21 +3123,29 @@ main.registerCommand({
     checkAndRemoveFiles()
   }
 
-  const packageJsonPath = files.pathJoin(appDir, 'package.json');
-  const packageJsonFile = files.readFile(packageJsonPath, 'utf8');
-  const packageJson = JSON.parse(packageJsonFile);
-
-  const mainJsPath =
-    packageJson?.meteor?.mainModule?.server
-      ? files.pathJoin(appDir, packageJson.meteor.mainModule.server)
-      : files.pathJoin(appDir, 'server', 'main.js');
-  const mainJs = files.readFile(mainJsPath);
-  const mainJsLines = mainJs.toString().split('\n');
-  const importLine = path
-    ? `import '${path}';`
-    : `import '/imports/api/${ scaffoldName }';`
-  const mainJsFile = [importLine, ...mainJsLines].join('\n');
-  files.writeFile(mainJsPath, mainJsFile);
+  if (importIntoServerMain || serverImportFile?.length > 0) {
```

Comment:
What happens if `importIntoServerMain` and `serverImportFile?.length > 0` are `false`?

Could you create some test scenarios for these?

Reply:
If both are false then no import is added anywhere which is a relevant use case.

I haven't found any tests for the generators. But I haven't looked too much.

Reply:
Could you confirm that? Then, we can include this in the next patch.

Reply:
Will do, but the nature of this change I think warrants this to be in v3.1 rather than a patch release.